### PR TITLE
@hippogriffic Here's a task to auto-build the svgstore file into the demo index.html

### PR DESCRIPTION
**Pull request by henrahmagix**
@incuna/js Please merge, ta!

### CODE REVIEW

**Comment by henrahmagix** [comment]
- [ ] Update README with demo instructions
- [ ] Update README with warning that changes to `index.html` will be overwritten by contents of `index.html.template` (because the former is still committed)

**Comment by Minglee01** [comment]
do you need to update the changelog?

**Comment by hippogriffic** [comment]
not yet, it's just merging into my branch, not master, changelog and readme will be updated there

File: Gruntfile.js [hunk 1]
```diff
@@ -53,27 +61,60 @@ module.exports = function (grunt) {
...
+        'process:demo'
+    ]);
 
-    grunt.registerTask('build', function () {
-        var tasks = [
-            'sass',
-            'svgstore'
-        ];
-        grunt.task.run(tasks);
+    grunt.registerMultiTask('process', 'Process a file wtih data', function () {
```
Comment (grahamgilchrist) [inline Gruntfile.js:106]: There is a convenient grunt module called `grunt-template` (https://github.com/mathiasbynens/grunt-template) which will do this without needing to register a custom task

**Comment by grahamgilchrist** [comment]
Looks ok to me 👍

**Comment by hippogriffic** [comment]
I'm going to merge this so I can keep working

File: Gruntfile.js (hunk 1 above)
Comment (henrahmagix) [inline Gruntfile.js:106]: That's the one I was looking for! I found `grunt-process`. Want me to change to use that?

**Comment by henrahmagix** [comment]
Lol I didn't realise how I'd named this pr

**Pull request merged**